tests/blockchain/eth/clients/test_mocked_clients.py

```diff
@@ -24,9 +24,10 @@ class MockGanacheProvider:
 
 
 class ChainIdReporter:
+
     # Support older and newer versions of web3 py in-test
     version = 5
-    chainID = 5
+    chainId = hex(5)
```

Comment:
Just out of curiosity: why hex here?

Reply:
Just following what geth nodes return as a type on this endpoint for mocking